restio/model.py

```diff
@@ -16,24 +16,28 @@ def _check_model_type(obj: Optional[BaseModel]):
 
 
 class ModelMeta:
-    __slots__ = ("init", "init_ignore_extra", "repr", "fields", "primary_keys")
+    __slots__ = ("init", "init_ignore_extra", "repr", "fields", "primary_keys", "alias")
 
     init: bool
     init_ignore_extra: bool
     repr: bool
     fields: Dict[str, Field]
     primary_keys: Dict[str, Field]
+    alias: Optional[str]
 
     def __init__(self):
         self.init = True
         self.init_ignore_extra = True
         self.repr = True
         self.fields = dict()
         self.primary_keys = dict()
+        self.alias = None
 
 
+# Meta attributes that don't get inherited from parent classes
+__model_meta_not_inherited__ = ("alias",)
```

Comment:
Since these variables are outside the classes, maybe they should be in all-caps?